libs/core/langchain_core/prompts/loading.py

```diff
@@ -72,13 +73,24 @@ def _load_examples(config: dict) -> dict:
     return config
 
 
+def _get_output_parsers_map() -> Dict[str, BaseOutputParser]:
+    parsers_map = {}
+    for parser in langchain_core.output_parsers.__all__:
+        parser_class = getattr(langchain_core.output_parsers, parser)
+        if hasattr(parser_class, '_type'):
+            if len(parser_class.__abstractmethods__) == 0:
+                parsers_map[parser_class()._type] = parser_class  # pylint: disable=abstract-class-instantiated
+    return parsers_map
+
+
 def _load_output_parser(config: dict) -> dict:
     """Load output parser."""
     if "output_parser" in config and config["output_parser"]:
+        output_parsers_map = _get_output_parsers_map()
         _config = config.pop("output_parser")
         output_parser_type = _config.pop("_type")
-        if output_parser_type == "default":
```

Comment:
we probably still want to support output_parser_type == "default"

Reply:
Ok, I will add it.

Reply:
Done @baskaryan